keps/sig-storage/625-csi-migration/README.md

```diff
@@ -540,6 +540,7 @@ Major milestones in the life cycle of a KEP should be tracked in `Implementation
 Major milestones for each in-tree plugin CSI migration:
 
 - 1.25
+  - Core CSI migration to GA
   - GCE PD CSI migration to GA
 - 1.24
   - AWS EBS CSI migration to GA
```

Comment:
I think AWS EBS wants to be GA in 1.25, not sure if it should be part of this PR.

Reply:
Done